app/src/main/java/de/westnordost/streetcomplete/settings/SettingsFragment.kt

```diff
@@ -100,6 +106,7 @@ class SettingsFragment : PreferenceFragmentCompat(),
     override fun onResume() {
         super.onResume()
         prefs.registerOnSharedPreferenceChangeListener(this)
+        updateQuestPreferenceSummary()
```

Comment:
Theoretically this data could change while the settings fragment did not move into the backround. I am thinking about the tablet-setup where you still see the settings menu on the left and the "submenu" on the right. I think currently this is not shown like that, but it would be more stable if you listened to changes on the VisibleQuestTypeSource on `onStart` and stopped listening on `onStop`. Whenever the visible quest type count changes, update the subtitle.